components/src/molecules/rating/Rating.stories.js

```diff
@@ -0,0 +1,66 @@
+import { storiesOf } from '@storybook/vue'
+import { action } from '@storybook/addon-actions'
+
```

Comment:
Could you add knob for `items` prop to edit this number dynamically?
Maybe also color picker knobs for `active-color` and `unchecked-color`?